mon-pix/mirage/factories/tutorial.js

```diff
@@ -0,0 +1,21 @@
+import { Factory } from 'ember-cli-mirage';
+import faker from 'faker';
+
+export default Factory.extend({
+
+  format() {
+    return 'page';
+  },
+
+  duration() {
+    return '00:05:00';
+  },
+
+  link() {
+    return faker.random.word();
```

Comment:
a priori on devrait utiliser une `url`
```suggestion
    return faker.internet.url();
```